Mathlib/Topology/Basic.lean

```diff
@@ -1359,6 +1359,12 @@ theorem IsOpen.preimage (hf : Continuous f) {t : Set Y} (h : IsOpen t) :
     IsOpen (f ⁻¹' t) :=
   hf.isOpen_preimage t h
 
+lemma Equiv.continuous_symm_iff (e : X ≃ Y) : Continuous e.symm ↔ IsOpenMap e := by
+  simp_rw [continuous_def, ← Set.image_equiv_eq_preimage_symm, IsOpenMap]
+
+lemma Equiv.isOpenMap_symm_iff (e : X ≃ Y) : IsOpenMap e.symm ↔ Continuous e := by
+  simp_rw [← Equiv.continuous_symm_iff, Equiv.symm_symm]
+
```

Comment:
Can you use one of these to golf `homeomorphOfContinuousOpen` in `Homeomorph.lean`?

Reply:
Yes, well spotted! Now I feel vindicated in including them in this PR :-)

Reply:
Oh sorry, I've only just noticed that in fact you suggested this four days ago!